store/driver/txn/unionstore_driver.go

```diff
@@ -0,0 +1,227 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package txn
+
+import (
+	"context"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/kv"
+	tikvstore "github.com/pingcap/tidb/store/tikv/kv"
+	"github.com/pingcap/tidb/store/tikv/unionstore"
+)
+
+// tikvBatchGetter wrapps unionstore.BufferBatchGetter as kv.BatchGetter.
```

Comment:
```suggestion
// tikvBatchGetter wraps unionstore.BufferBatchGetter as kv.BatchGetter.
```

And for all same comments in the same file..